src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx

```diff
@@ -43,7 +43,8 @@ credentials = {
 }
 ```
 
-Using the default options would not cause the _object_ `credentials` to be redacted in its entirety; rather, all of its entries would be subject to scrubbing. So `password` would be redacted by default, and adding `cats`, `username`, and/or `lastLogin` to the list of additional fields would cause those values to be redacted as well.
+Using the default options would cause the _object_ `credentials` to be set to `null`, and the object itself will not be entered. To prevent this from happening the keyname can be added to Safe Fields (`credentials` in this case), which will leave the `credentials` object intact, but all of its entries would be subject to scrubbing.
+So `password` would be redacted by default, and adding `cats`, `username`, and/or `lastLogin` to the list of additional fields would cause those values to be redacted as well.
```

Comment:
```suggestion
Using the default options will cause the _object_ `credentials` to be set to `null`, and the object itself will not be entered. To prevent this from happening add the keyname to Safe Fields (`credentials` in this case). This will leave the `credentials` object intact, while making all of its entries subject to scrubbing.
So `password` will be redacted by default, and adding `cats`, `username`, and/or `lastLogin` to the list of additional fields will cause those values to be redacted as well.
```